### PR TITLE
Improve logging of module imports

### PR DESCRIPTION
**Pull request by adrian-prantl**

### CODE REVIEW

**Comment by adrian-prantl** [comment]
@swift-ci test and merge

File: source/Plugins/Language/Swift/SwiftHashedContainer.cpp [hunk 1]
```diff
@@ -851,7 +851,7 @@ LegacyNativeHashedStorageHandler::LegacyNativeHashedStorageHandler(
   m_ptr_size = m_process->GetAddressByteSize();
 
   auto bucketCount_sp =
-    m_nativeStorage->GetChildAtNamePath({g_bucketCount, g__value});
+LegacyNativeHashedStorageHandler    m_nativeStorage->GetChildAtNamePath({g_bucketCount, g__value});
```
Comment (dcci) [inline source/Plugins/Language/Swift/SwiftHashedContainer.cpp]: Are those changes intended?

**Comment by adrian-prantl** [comment]
@swift-ci test and merge

**Comment by adrian-prantl** [comment]
@swift-ci test and merge

**Pull request merged**